catalogue_pipeline/id_minter/src/main/scala/uk/ac/wellcome/platform/idminter/models/JsonKeyPrefixGenerator.scala

```diff
@@ -4,7 +4,7 @@ import io.circe.Json
 import uk.ac.wellcome.storage.s3.KeyPrefixGenerator
 
 class JsonKeyPrefixGenerator extends KeyPrefixGenerator[Json] {
-  override def generate(obj: Json): String = {
+  override def generate(id: String, obj: Json): String = {
```

Comment:
This looks like it has key prefix generator fixes mixed in that we want to replace with your new storage stuff?